detection-rules/attachment_emotet_heavily_padded_doc_in_zip.yml

```
@@ -0,0 +1,32 @@
+name: "Attachment: Emotet heavily padded doc in zip file"
+description: |
+  Detects a potential Emotet delivery method using padded .doc files that compress into small zip files. 
+  Contents may include Red Dawn templates exceeding 500MB.
+references:
+  - "https://twitter.com/Cryptolaemus1/status/1633099154623803394"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and any(attachments, 
+    .file_extension == "zip"
+    and any(file.explode(.), 
+      .depth == 0 and .size < 1000000
+      and not .depth > 0
```

```suggestion
      not .depth > 0
      and .depth == 0 and .size < 1000000
```

I could see reversing that order to bail if there are depths greater than 0?

---

sounds like you want to do two checks?